shared.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * This config is shared between the default and react configs
+ */
+module.exports = {
+  plugins: [
+    "jest",
+    "jsdoc",
+    "sort-keys-fix",
+    "sort-destructure-keys",
+    "node",
+    "@typescript-eslint",
+  ],
+  rules: {
+    "brace-style": ["error", "stroustrup"],
+    "comma-dangle": ["error", "always-multiline"],
+    curly: ["error", "multi-line", "consistent"],
+    "no-unused-vars": "off",
```

Comment:
Why are we turning this one off?

Reply:
[As per these docs](https://typescript-eslint.io/rules/no-unused-vars/) because we've got the typescript version below. I've just checked and `@typescript-eslint/no-unused-vars` does still lint non-ts files.